packages/thrift-integration/src/client/connections/HttpConnection.spec.ts

```diff
@@ -88,24 +88,30 @@ describe('HttpConnection', () => {
             )
         })
 
-        it('should corrently handle a void service client request', async () => {
+        it('should correctly handle a void service client request', async () => {
             return client.ping().then((response: any) => {
                 expect(response).to.equal(undefined)
             })
         })
 
-        it('should corrently handle a service client request that returns a struct', async () => {
+        it('should correctly handle a service client request that returns a struct', async () => {
             return client.getStruct(5).then((response: ICommonStruct) => {
-                expect(response).to.equal({
-                    code: { status: new thrift.Int64(0) },
-                    value: 'test',
-                })
+                expect(response).to.equal(
+                    {
+                        code: { status: new thrift.Int64(0) },
+                        value: 'test',
+                    },
+                    { skip: ['_annotations', '_fieldAnnotations'] },
```

Comment:
Why do we need to skip comparing these fields now? (in this file and in others in the PR)

Reply:
There were some changes in the `@hapi/code` library (test matchers, https://github.com/hapijs/code) that seem to make this not work the way it was before - specifically object comparisons where one is missing keys that the other has. I think that it may have been caused by [this change](https://github.com/hapijs/code/commit/91802a7addbfd050c136ae4c2d9d8f2a7e602c93)?

Anyways, the tests were failing because those two keys were present in the resulting object but not in the object here in the test. I couldn't figure out a better way to do this other than skipping the problematic properties (the `includes` function didn't seem to do it). In Jest I'd just use `expect.objectContaining` but I couldn't find similar functionality in `@hapi/code`.